release/README.md

```diff
@@ -91,10 +91,10 @@ gcloud container clusters get-credentials bank-of-anthos-release --zone us-centr
 ```
 1. ***[Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) enabled production cluster***
 
-Currently the `bank-of-anthos-release` cluster has [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) enabled. Thus, when deploying to this cluster the pod service account (`boa-ksa-master`) used by _Workload Identity_ must be used as the serviceAccount in the manifests.
+Currently the `bank-of-anthos-release` cluster has [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) enabled. Thus, when deploying to this cluster the pod service account (`boa-ksa-release`) used by _Workload Identity_ must be used as the serviceAccount in the manifests.
 
-Follow steps 3 and 4 of the [workload identity setup](https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/master/docs/workload-identity.md) with the following config values to deploy into production:
-- `boa-ksa-master` as the `KSA_NAME`
+Follow steps 3 and 4 of the [workload identity setup](https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/main/docs/workload-identity.md) with the following config values to deploy into production:
+- `boa-ksa-release` as the `KSA_NAME`
```

Comment:
let's be careful w/ this since I'm guessing there's already a KSA called `boa-ksa-master` on our prod cluster @bourgeoisor  - so we may have to redo that part of the WI setup

Reply:
Yep, talked to Christine about this before she made those changes, I'll deploy WI based on this!